### PR TITLE
git-init: add --initial-branch example

### PR DESCRIPTION
**Pull request by navarroaxel**
- [ ] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/git-init.md [hunk 1]
```diff
@@ -7,6 +7,10 @@
 
 `git init`
 
+- Initialize a repository with a specific name for the initial branch:
```
Comment (bl-ue) [inline pages/common/git-init.md:10]: Sounds a bit more to-the-point.
```suggestion
- Initialize a repository with the specifed name for the initial branch:
```

**Comment by bl-ue** [comment]
@navarroaxel do you mind updating the [Spanish translation](https://github.com/tldr-pages/tldr/blob/pages/git-init/pages.es/common/git-init.md) too?

File: pages.es/common/git-init.md [hunk 2]
```diff
@@ -7,7 +7,11 @@
 
 `git init`
 
-- Inicializa un repositio usando SHA256 como hash del objeto (requiere la versión 2.29+ de Git):
```
Comment (bl-ue) [inline pages.es/common/git-init.md:10]: Is `repositio` just a typo, or a different declension?

Comment (navarroaxel) [reply pages.es/common/git-init.md:10]: a typo.

Comment (bl-ue) [reply pages.es/common/git-init.md:10]: Good to know. 👍🏻

**Review by sbrl — APPROVED** [review]
Thanks, @navarroaxel!

**Review by CleanMachine1 — APPROVED** [review]

**Review by bl-ue — APPROVED** [review]

**Pull request merged**